logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -208,13 +210,28 @@ actual class CallManagerImpl(
             }
         }
 
-        // TODO(calling): Network Quality handler
+        // Network Quality
+        scope.launch {
+            withCalling {
+                val onNetworkQualityChanged = OnNetworkQualityChanged()
+
+                wcall_set_network_quality_handler(
+                    inst = deferredHandle.await(),
+                    wcall_network_quality_h = onNetworkQualityChanged,
```

Comment:
We might need to call `keepingStrongReference()` here once #536 is merged.

Reply:
Yep! Just waiting for it to be merged.